lib/ex_fizz_buzz_enterprise/input_worker.ex

```diff
@@ -22,13 +21,24 @@ defmodule ExFizzBuzzEnterprise.InputWorker do
     GenServer.cast(__MODULE__, :next)
   end
   
+  def update(state) do
+    GenServer.call(__MODULE__, {:update, state})
```

Comment:
Like this one - this one is perfect in terms of consistency.  Only nitpick is I might call it `put_state` or something just because `update` usually implies you're going to calculate a new state as a function of the old state, like `update(fn(old_state) -> do_stuff_with(old_state) end)`.  That's really nitpicky though.